.github/workflows/spdx.yml

```diff
@@ -0,0 +1,42 @@
+name: Update license data
+
+on:
+  schedule:
+    - cron: '*/60 * * * *'
```

Comment:
Maybe run it less frequently? Like once per 8 hours?

Reply:
With #8023 this could conceivably be as infrequent as once a day?

Reply:
You mean `"$(brew --repo)/Library/Homebrew"`?